src/test/scala/com/typesafe/npm/NpmSpec.scala

```diff
@@ -15,7 +15,7 @@ class NpmSpec extends Specification with NoTimeConversions {
   def withEngine[T](block: ActorRef => T): T = {
     val system = ActorSystem("test-system", classLoader = Some(this.getClass.getClassLoader))
     try {
-      val engine = system.actorOf(Trireme.props(), "engine")
+      val engine = system.actorOf(Node.props(), "engine")
```

Comment:
Sorry I mean `sbt test` fails when having node 7.9.0 installed locally (has nothing to do with my requested change)